languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -28745,4 +28607,551 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rulegroup>
     </category>
 
+   <!-- ====================================================================== -->
+   <!-- Hilefestellung für Kommasetzung -->
+   <!-- ====================================================================== -->
+   <category id="HILFESTELLUNG_KOMMASETZUNG" name="Hilefestellung für Kommasetzung">
+		
+	    <rulegroup id="KOMMA_INFINITIVGRUPPEN" name="Komma vor Infinitivgruppen mit 'als', 'anstatt', 'außer', 'ohne', 'statt', 'um'">
+		    <url>http://www.duden.de/sprachwissen/rechtschreibregeln/komma</url>
```

Comment:
Hi! I am the author of the rulegroup "KOMMA_VOR_INFINITIVGRUPPE" I have done a quick check of "KOMMA_INFINITIVGRUPPE". Compared to the exisiting rulegroup your rules trigger some false alarms, for instance:

- Es scheint sich um einen Fehler zu handeln.
- Er braucht sich um diese Sache nicht zu kümmern.
- Jeden Tag gehe ich um halb acht aus dem Haus und fange um zweiundzwanzig Uhr zu arbeiten an.
- Es ist Ihre Pflicht als Bürger aufzupassen.
- Ihr Plan scheint besser als meiner zu sein.

( Infinitivgruppen werden im Allgemeinen nicht durch Komma abgetrennt, wenn die Infinitivgruppe von einem Hilfsverb oder von „brauchen", „pflegen", „scheinen"abhängig ist.)

Maybe you should add some antipatterns.